### PR TITLE
Displaying Collaborators in the extension

### PR DESCRIPTION
**Pull request by stkem**
Sill making some small CSS tweaks, but structurally reviewable. 
@andrewconner @2is10

### CODE REVIEW

**Comment by stkem** [comment]
Happy to show my local version

File: packrat/scripts/keep_box.js [hunk 1]
```diff
@@ -1096,9 +1096,24 @@ k.keepBox = k.keepBox || (function () {
...
+        var collab = collabs.shift();
+        names = names + ', ' + collab.firstName;
+      }
+      if (collabs.length > 1) {
+        names = names + ', ' + collabs.length + ' others';
+      } else if (collabs.length === 1) {
+        names = names + ', ' + collabs[0].firstName;
+      }
+      return names;
+    }
```
Comment (2is10) [inline packrat/scripts/keep_box.js:1112]: Might want to consider just listing all available collaborator names, comma-separated, and add `text-overflow: ellipsis`. I don’t think the total number is very important in this context. As you know, a character limit is a heuristic that can both overly and “underly” restrict text width.

Also, in other places, the server only sends `BasicUser` info for a sampling of the collaborators, along with the total number, in order to scale nicely to a large set of collaborators. We might want to follow that pattern here too.

**Comment by 2is10** [comment]
:+1: nicely done!

**Comment by stkem** [comment]
@andrewconner This is ready to go. Let me know what you would like to do about release.
(I'll likely make some CSS tweaks after the fact, but that is not a blocker)

**Comment by andrewconner** [comment]
Great. Let's leave it open until site is ready.

**Pull request merged**